Assets/VRTK/Source/Scripts/Interactions/Interactables/Controllables/Artificial/VRTK_ArtificialPusher.cs

```diff
@@ -67,6 +67,11 @@ public override float GetNormalizedValue()
             return VRTK_SharedMethods.NormalizeValue(GetValue(), originalLocalPosition[(int)operateAxis], PressedPosition()[(int)operateAxis]);
         }
 
+        //Has no implementation as pushers reset automatically anyway.
```

Comment:
Put proper docs in here otherwise the generator won't pick it up.

```
/// <summary>
/// The SetValue method is not implemented as the pusher resets automatically.
/// </summary>
/// <param name="value">Not used.</param>
```